addons/radar/XEH_preInit.sqf

```diff
@@ -466,6 +466,18 @@ if (isClass(configfile >> "CfgPatches" >> "diwako_dui_buddy")) then {
     diwako_dui_namelist_only_buddy_icon = false;
 };
 
+[
+    QGVAR(icon_priority_setting)
+    ,"LIST"
+    ,[localize "STR_dui_namelist_icon_priority", localize "STR_dui_namelist_icon_priority_desc"]
+    ,[CBA_SETTINGS_CAT, _curCat]
+    ,[[0, 1], [localize "STR_dui_namelist_icon_priority_setting0", localize "STR_dui_namelist_icon_priority_setting1"], 1]
+    ,false
```

Comment:
Unsure if this setting has to be enforced globally or not.

Reply:
While I think the icon style should be a client side setting, what the icon represents and on which position in the list it should be is not. I would say this should be a globally enforced setting.

Reply:
Changed.